bugsnag-android-core/src/androidTest/java/com/bugsnag/android/ConcurrentCallbackTest.java

```diff
@@ -33,7 +33,8 @@ public void tearDown() throws Exception {
 
     @Test
     public void testClientNotifyModification() throws Exception {
-        final Collection<BeforeNotify> beforeNotifyTasks = client.config.getBeforeNotifyTasks();
+        Configuration config = (Configuration) client.getConfiguration();
```

Comment:
We cast `BugsnagConfiguration` to `Configuration` for the purposes of testing only.